File: Beginner/Exer. 1009/main.c
#include <stdio.h>

int main(int argc, char *argv[]) {
    char name[256];
    double salary;
    double sales;
    
    scanf("%s", name);
    scanf("%lf", &salary);
    scanf("%lf", &sales);
    
    
    printf("TOTAL = R$ %.2lf\n", (salary + (sales * 0.15)));
    return 0;
}

File: Beginner/Exer. 1009/main.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        String name = new String();
        double salary, sales;

        Scanner scanner = new Scanner(System.in);
        name = scanner.nextLine();
        salary = scanner.nextDouble();
        sales = scanner.nextDouble();
        scanner.close();

        System.out.printf("TOTAL = R$ %.2f\n", (salary + (sales * 0.15)));
    }
}